student.rb

```diff
@@ -1,9 +1,12 @@
-retuire_relative 'person'
+require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name: name, parent_permission: parent_permission)
     @classroom = classroom
+    classroom.add_student(self)
```

Comment:
- [ Required ] I think it would be better to factor out this part of your code into a standalone `setter` method as specified in the project requirements. Setting the classroom of a student only in the constructor and not providing a setter method makes it impossible for someone to update the classroom of the student after the `instance` of the class has been created. If you create a setter method for setting the classroom, then the classroom attribute can be updated even after the `instance` of the class has been created. I would like to suggest that you refactor your code as shown below.

## Example code
> I think according to the project requirements the classroom attribute should be read-only and the only way to set it should be through the setter method as shown below, so that when setting the classroom for a student it also adds it to the classrooms' students.

```ruby
require_relative 'person'

class Student < Person
  attr_reader :classroom

  def initialize(age, name, parent_permission)
    super(age, name, parent_permission: parent_permission)
  end

  def play_hooky
    '¯\(ツ)/¯'
  end

  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```

## Project requirements
![Screenshot from 2023-07-27 23-54-32](https://github.com/luigirazum/library-manager/assets/16392046/dcd739ed-04b3-4eb4-a03f-dfd065dabd1d)

Reply:
Hi @Felix45 
Thank you for your comments add detailed instruction, the issue has been resolved.